### PR TITLE
fix: aminos 🇮🇳 Updated 

### PR DESCRIPTION
**Pull request by amino19**
IssueNumber same, Link Updated.
 
_Ready to Merge!_ :slightly_smiling_face: 

Related to #582

### CODE REVIEW

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket:

**Comment by amino19** [comment]
@kaiwalyakoparkar : Something is failing! :confused:

**Review by deephunt3r — APPROVED** [review]
LGTM 💯

**Comment by kaiwalyakoparkar** [comment]
I guess the prettier is failing. :/

**Comment by amino19** [comment]
@kaiwalyakoparkar : Yep! It is! :confused: 
I think, I have to rework!

**Comment by amino19** [comment]
@praveenscience : Did you like to give any suggestion, I am using `prettier` in vsc, but there I didn't get any error/bug, but here it's caught ! 
idk what! :confused:

**Review by schmelto — APPROVED** [review]

File: data.json [hunk 1]
```diff
@@ -914,7 +914,7 @@
   {
     "name": "aminos",
     "githubUsername": "amino19",
-    "imageUrl": "https://user-images.githubusercontent.com/75872316/118384765-6dad0480-b626-11eb-85ce-a6af24c19181.png",
+    "imageUrl": "https://user-images.githubusercontent.com/75872316/125152497-91924f80-e16a-11eb-8366-8d5c3d26df10.jpg",
```
Comment (Panquesito7) [inline data.json:917]: I'm guessing the filename here should be `.png` for the Prettier to stop failing?

Comment (amino19) [reply data.json:917]: @Panquesito7 : Yes actually I also thought that previously, but some members said it occurs sometimes.
Also, there're more `.jpg` image inside!

**Comment by Panquesito7** [comment]
@schmelto, maybe try re-running the workflow? Thanks.

**Pull request merged**